pkg/utils/gardener/shoot.go

```diff
@@ -279,6 +280,13 @@ func IsShootProjectInternalSecret(secretName string) (string, bool) {
 	return "", false
 }
 
+// ComputeManagedShootIssuerSecretName returns the name that should be used for
+// storing the service account public keys of a shoot's kube-apiserver
+// in the gardener-system-shoot-issuer namespace in the Garden cluster.
+func ComputeManagedShootIssuerSecretName(projectName string, shootUID types.UID) string {
+	return fmt.Sprintf("%s--%s", projectName, shootUID)
```

Comment:
```suggestion
	return projectName + "--" + shootUID
```

Concatenation looks like to be three times faster than string formatting, ref:

```go
package benchmark_test

import (
	"fmt"
	"testing"
)

var (
	projectName = "foo"
	uid         = "da22b419-b463-40d2-b0a7-480e32ef86a5"
)

func concat(project, uuid string) string {
	return project + "--" + uuid
}

func format(project, uuid string) string {
	return fmt.Sprintf("%s--%s", project, uuid)
}

func BenchmarkConcat(b *testing.B) {
	for n := 0; n < b.N; n++ {
		_ = concat(projectName, uid)
	}
}

func BenchmarkFormat(b *testing.B) {
	for n := 0; n < b.N; n++ {
		_ = format(projectName, uid)
	}
}
```

```shell
$ go test -bench=. -benchtime=10s
goos: darwin
goarch: amd64
pkg: github.com/gardener/gardener/benchmark
cpu: Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
BenchmarkConcat-16    269962924         43.68 ns/op
BenchmarkFormat-16      76994737               161.0 ns/op
PASS
ok      github.com/gardener/gardener/benchmark  28.979s
```